Microsoft.Toolkit.HighPerformance/Enumerables/RefEnumerable{T}.cs

```diff
@@ -9,9 +9,9 @@
 using System.Runtime.InteropServices;
 #endif
 using Microsoft.Toolkit.HighPerformance.Helpers.Internals;
-#if SPAN_RUNTIME_SUPPORT
 using Microsoft.Toolkit.HighPerformance.Memory.Internals;
-#else
+
+#if !SPAN_RUNTIME_SUPPORT
```

Comment:
Nit: remove this blank line.
Also could you do the same in `ReadOnlyRefEnumerable<T>` as well? 🙂
```suggestion
#if !SPAN_RUNTIME_SUPPORT
```